doc/howto/storage_volumes.md

```diff
@@ -189,7 +192,7 @@ In some cases, it is also possible to reduce the size of a storage volume.
 
 To resize a storage volume, set its size configuration:
```

Comment:
```suggestion
To resize the quota of a storage volume, set its size configuration:
```

Reply:
Per https://github.com/canonical/lxd/pull/14578#issuecomment-2523201351 - indicate that quota is another word for size in LXD docs.

Reply:
(also added https://github.com/canonical/lxd/pull/14604 to update the rest of the docs)